src/App.js

```diff
@@ -1,12 +1,161 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import Mainbutton from "./components/Mainbutton";
+import Input from "./components/Input";
+import Fruitcounter from "./components/Fruitcounter";
+import emoji from "emoji-dictionary";
+
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+    const [strawberryCounter, setStrawberryCounter] = useState(0);
+    const [bananaCounter, setBananaCounter] = useState(0);
+    const [appleCounter, setAppleCounter] = useState(0);
+    const [kiwiCounter, setKiwiCounter] = useState(0);
+
+    function decreaseCounter(counter, setCounter) {
+        if (counter > 0) {
+            setCounter(counter - 1)
+        }
+    }
```

Comment:
Het is netter (en gebruikelijker) om de button door middel van de `disabled` property uit te zetten, zodat de counter niet ónder de 0 kan gaan. Dan hoef je deze functie niet te maken 😉